experimenter/experimenter/nimbus_ui_new/views.py

```diff
@@ -215,3 +217,23 @@ class MetricsUpdateView(NimbusExperimentViewMixin, RequestFormMixin, UpdateView)
     def form_valid(self, form):
         super().form_valid(form)
         return self.render_to_response(self.get_context_data(form=form))
+
+
+class SubscribeView(NimbusExperimentViewMixin, RequestFormMixin, UpdateView):
+    form_class = SubscribeForm
+    template_name = "nimbus_experiments/subscribers_list.html"
+
+    def form_valid(self, form):
+        form.user = self.request.user
```

Comment:
This should be handled by the `RequestFormMixin` that's already included in this view?

Reply:
getting error without it

Reply:
I tried without this too